snapteld.go

```diff
@@ -438,7 +438,7 @@ func action(ctx *cli.Context) error {
 							log.WithFields(
 								log.Fields{
 									"block":       "main",
-									"_module":     "snapd",
+									"_module":     "snapteld",
```

Comment:
Instead of changing these all from snapd->snapteld, would it make sense to just instantiate a logger with the module set in one place? Or at least use a moduleName var?